projects/kit/components/select/select-option/select-option.component.ts

```diff
@@ -1,18 +1,30 @@
 import {ChangeDetectionStrategy, Component, computed, inject} from '@angular/core';
 import {tuiDirectiveBinding, tuiIsPresent} from '@taiga-ui/cdk/utils/miscellaneous';
-import {TuiOptionWithValue} from '@taiga-ui/core/components/data-list';
+import {
+    TuiDataListComponent,
+    TuiOptionWithValue,
+} from '@taiga-ui/core/components/data-list';
 import {TuiScrollIntoView} from '@taiga-ui/core/components/scrollbar';
-import {TuiIcons} from '@taiga-ui/core/directives/icons';
 import type {TuiItemsHandlers} from '@taiga-ui/core/directives/items-handlers';
 import {TUI_ITEMS_HANDLERS} from '@taiga-ui/core/directives/items-handlers';
-import {TUI_COMMON_ICONS} from '@taiga-ui/core/tokens';
+import {TuiCheckbox, tuiCheckboxOptionsProvider} from '@taiga-ui/kit/components/checkbox';
 import {tuiInjectValue} from '@taiga-ui/kit/utils';
 
 @Component({
     standalone: true,
-    template: '',
+    imports: [TuiCheckbox],
+    template: `
+        <input
+            tuiCheckbox
+            type="checkbox"
+            class="t-check"
+            [checked]="selected()"
+            [size]="datalist.size === 'l' ? 'm' : 's'"
```

Comment:
Too little checkmark icon for `L`-size ?

<img width="1484" alt="demo" src="https://github.com/user-attachments/assets/d6d38afa-c48c-49fd-ab5f-85d3803fcabe" />

<hr />

<img width="1205" alt="spec" src="https://github.com/user-attachments/assets/c838d47b-988b-42aa-ba5a-80d20130b69c" />